capstone/src/main/resources/templates/profile.html

```diff
@@ -73,6 +73,18 @@
                     </select>
                     <label>Description:</label>
                     <input type="text" name="description" id="org-description" placeholder="Description of your organization.."></input>
+                    <label>Organization Type:</label>
+                    <select name="org-type" id="org-type">
+                        <option value="" disabled selected hidden>Choose organization type</option>
+                        <option value="academic">Academic</option>
```

Comment:
You can avoid hard-coding and duplicating the list of organization types by either creating an array in JavaScript shared across the two pages or storing  the types in a special OrgType table in Datastore.